controls/barcode/2d-barcodes/qrcode-overview.md

```diff
@@ -17,31 +17,31 @@ A QR code uses four standardized encoding modes to efficiently store data&mdash;
+* Alignment Pattern&mdash;The alignment pattern is only included in the rendered QR code in version 2 and above. Its purpose is to allow the decoder to scan a skewed image, and convert it to the virtual grid of black and white modules, representing the encoded data. The alignment pattern is made of concentric squares, like the finder patterns, with the center being a single black module.
+* Timing Pattern&mdash;The timing pattern is an alternating stripe of black and white modules, starting at the lower left corner of the upper right Finder Pattern, going horizontally to the upper left finder pattern and then going vertically to the lower left finder pattern.
+* Format Data&mdash;The format data is information, about the Masking rule used in the QR Code, along with the error correction level. When the data in the QR code is encoded, some of the modules are inverted following a predefined rule to improve readability and ensure that has no big clusters of same-colored modules. This process is called masking, and the masking information is included in the format data, to alert the decoder that some modules have been inverted.
 
   The format data is encoded in 15 bits. One full copy of the format data is located around the upper left finder pattern. A second copy, divided into 7 and 8 bits, is located next to the other two finder patterns.
 
-* **Version Data**&mdash;The version data includes information on which version the QR code is. This data is encoded into 18 modules, in a 6 by 3 matrix. Two copies of the version data matrix are included in the QR code: one next to the upper right finder pattern and the other next to the lower left one.
-* **Blank Space**&mdash;Additionally, around each QR code, there is an obligatory 4-modules-wide white space area:
+* Version Data&mdash;The version data includes information on which version the QR code is. This data is encoded into 18 modules, in a 6 by 3 matrix. Two copies of the version data matrix are included in the QR code: one next to the upper right finder pattern and the other next to the lower left one.
+* Blank Space&mdash;Additionally, around each QR code, has an obligatory 4-modules-wide space area:
 
   ![qrcode blank space](images/barcode-2d-barcodes-qrcode-overview004.png)
 
-* **Data**&mdash;The data occupies all available modules that are not occupied by any of the formatting data segments. If the data is less than the capacity of the remaining modules, it is padded to ensure that all modules are used. Additionally, the data consists of the actual encoded data, entered by the user, and the error correction bits, calculated on that data.
+* Data&mdash;The data occupies all available modules that are not occupied by any of the formatting data segments. If the data is less than the capacity of the remaining modules, it's padded to ensure that all modules are used. Additionally, the data consists of the encoded data, entered by the user, and the error correction bits, calculated on that data.
```

Comment:
Don't modify every occurrence of "It is" that the ReDox tool suggests. This suggestion is valid only for cases where "it is" is used as a placeholder and serves a purely grammatical function without adding meaning, for example:

- It is believed by some people that the Earth is flat -> Some people believe that the Earth is flat.
- It is a common mistake to swap "they're" with "their" -> Swapping "they're" with "their" is a common mistake.

The same rule is true for "there is" and "there are". See the [Style Guide](https://docs.telerik.com/style-guide/using-simple-non-ambiguous-language#using-simple-non-ambiguous-language) for more examples.
 
In the sentence that you modify, "it" is a pronoun that refers to "the data" and this loads the pronoun "it" with a specific meaning (the data). So in this case, "it is" is not serving a purely grammatical function, it has meaning.

```suggestion
* Data&mdash;The data occupies all available modules that are not occupied by any of the formatting data segments. If the data is less than the capacity of the remaining modules, it is padded to ensure that all modules are used. Additionally, the data consists of the encoded data, entered by the user, and the error correction bits, calculated on that data.
```